ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -1,44 +1,44 @@
 !
-{%     if logging.console is defined and logging.console is not none %}
+{%     if logging.console is arista.avd.defined %}
 logging console {{ logging.console }}
 {%     endif %}
-{%     if logging.monitor is defined and logging.monitor is not none %}
+{%     if logging.monitor is arista.avd.defined %}
 logging monitor {{ logging.monitor }}
 {%     endif %}
-{%     if logging.buffered is defined and logging.buffered is not none %}
+{%     if logging.buffered.size is arista.avd.defined and logging.buffered.level is arista.avd.defined %}
 logging buffered {{ logging.buffered.size }} {{ logging.buffered.level }}
 {%     endif %}
-{%     if logging.trap is defined and logging.trap is not none %}
+{%     if logging.trap is arista.avd.defined %}
 logging trap {{ logging.trap }}
 {%     endif %}
-{%     if logging.format is defined and logging.format is not none %}
-{%        if logging.format.timestamp is defined and logging.format.timestamp == 'high-resolution' %}
+{%     if logging.format is arista.avd.defined %}
+{%        if logging.format.timestamp is arista.avd.defined('high-resolution') %}
 logging format timestamp high-resolution
 {%        endif %}
-{%        if logging.format.hostname is defined and logging.format.hostname is not none %}
-{%              if logging.format.hostname is defined and logging.format.hostname == 'fqdn' %}
+{%        if logging.format.hostname is arista.avd.defined %}
+{%              if logging.format.hostname is arista.avd.defined('fqdn') %}
 logging format hostname fqdn
-{%              elif logging.format.hostname is defined and logging.format.hostname == 'ipv4' %}
+{%              elif logging.format.hostname is arista.avd.defined('ipv4') %}
 logging format hostname ipv4
 {%              endif %}
 {%        endif %}
-{%        if logging.format.sequence_numbers is defined and logging.format.sequence_numbers == true %}
+{%        if logging.format.sequence_numbers is arista.avd.defined(true) %}
 logging format sequence-numbers
 {%        endif %}
 {%     endif %}
-{%     if logging.source_interface is defined and logging.source_interface is not none %}
+{%     if logging.source_interface is arista.avd.defined %}
 logging source-interface {{ logging.source_interface }}
 {%     endif %}
-{%     if logging.vrfs is defined and logging.vrfs is not none  %}
+{%     if logging.vrfs is arista.avd.defined %}
 {%          for vrf in logging.vrfs | arista.avd.natural_sort %}
-{%              if logging.vrfs[vrf].source_interface is defined and logging.vrfs[vrf].source_interface is not none and vrf is ne 'default' %}
+{%              if logging.vrfs[vrf].source_interface is arista.avd.defined and vrf is ne 'default' %}
 logging vrf {{ vrf }} source-interface {{ logging.vrfs[vrf].source_interface }}
-{%              elif logging.vrfs[vrf].source_interface is defined  and vrf == 'default' %}
+{%              elif logging.vrfs[vrf].source_interface is arista.avd.defined and vrf == 'default' %}
```

Comment:
{%              elif logging.vrfs[vrf].source_interface is arista.avd.defined("default") %}